lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -41,7 +41,7 @@ function addToUsersBucket(canonicalID, bucketName, bucketMD, log, cb) {
             usersBucket : oldUsersBucket;
         return metadata.putObjectMD(usersBucketBeingCalled, key,
             omVal, {}, log, err => {
-                if (err && err.NoSuchBucket) {
+                if (err && err.is.NoSuchBucket) {
```

Comment:
We could do something like that everywhere we have the `err && err.is` pattern:

```suggestion
                if (err?.is.NoSuchBucket) {
```

Reply:
ah, linter doesn't allow TT

Reply:
fixed linter